jwst/photom/photom.py

```diff
@@ -575,8 +580,6 @@ def calc_miri(self, ftab):
                 self.input.meta.bunit_data = 'DN/s'
                 self.input.meta.bunit_err = 'DN/s'
 
-        return
```

Comment:
Also strictly unrelated but no need to return here. Python returns None by default on exit anyway.